webapp/src/Service/ScoreboardService.php

```diff
@@ -1103,16 +1096,20 @@ protected function getScorecache(Contest $contest, ?Team $team = null): array
      * Get the rank cache for the given team.
      * @throws NonUniqueResultException
      */
```

Comment:
```suggestion
     returns RankCache[]
     */
```
(I suspect that's the return type, please verify)

I wonder why there is no test complaining about this

Reply:
I assume you meant `@return RankCache[]`?

> I wonder why there is no test complaining about this

Why would tests complain about a missing non-concrete return type?

Reply:
> I assume you meant `@return RankCache[]`?
> 
Yes, the `returns` should be `return` and I suspect the function returns an array of `RankCache`.

> > I wonder why there is no test complaining about this
> 
> Why would tests complain about a missing non-concrete return type in a docstring?

I thought we had PHPStan configured to also complain if the return type is not 100% clear as it uses it for further static analysis.

Reply:
Ok, that's done. (Never seen such a PHPstan comment though.)